packages/charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -164,6 +169,38 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
     });
   };
 
+  private _getChartDataText(data: IChartProps): string {
+    const chartDataMode = this.props.chartDataMode || ChartDataMode.default;
+    const x = data!.chartData![0].horizontalBarChartdata!.x;
+    const y = data!.chartData![0].horizontalBarChartdata!.y;
+
+    switch (chartDataMode) {
+      case ChartDataMode.default:
+        return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+      case ChartDataMode.fraction:
+        return x + '/' + y;
+      case ChartDataMode.percentage:
+        const dataRatio = Math.round((x / y) * 100);
+        return dataRatio + '%';
+    }
+  }
+
+  private _createBenchmark(data: IChartProps): JSX.Element {
+    if (data.chartData![0].data) {
+      const totalData = data.chartData![0].horizontalBarChartdata!.y;
+      const benchmarkData = data.chartData![0].data;
+      const benchmarkRatio = Math.round(((benchmarkData ? benchmarkData : 0) / totalData) * 100);
+
+      const benchmarkStyles: IStyle = {
+        marginLeft: 'calc(' + benchmarkRatio + '% - 8px)'
+      };
+
+      return <div className={mergeStyles(this._classNames.triangle, benchmarkStyles)} />;
```

Comment:
We can't add marginLeft style in triangle class via the "styles" prop. Because triangle class apply to the whole HorizontalBarChart component. HorizontalBarChart component contains a list of barchart(datapoint array) and each barchart have different marginLeft value for benchmark triangle. If we set marginLeft style in triangle class, all barcharts will have the same marginLeft value for benchmark triangle.

![image](https://user-images.githubusercontent.com/12230173/58232219-923db100-7d6b-11e9-9aa6-5fcdd07b7db3.png)

Reply:
Remove mergeStyles and change it to 
<div className={this._classNames.triangle} style={benchmarkStyles} />;

Reply:
Remove mergeStyles and change it to `<div className={this._classNames.triangle} style={benchmarkStyles} />`